client/me/concierge/shared/confirmation.js

```diff
@@ -11,35 +11,33 @@ import React, { Component } from 'react';
 import Button from 'components/button';
 import Card from 'components/card';
 import FormattedHeader from 'components/formatted-header';
-import { localize } from 'i18n-calypso';
 
-class ConfirmationStep extends Component {
+class Confirmation extends Component {
 	render() {
-		const { site, translate } = this.props;
+		const { site, confirmationButton, confirmationDescription, confirmationTitle } = this.props;
```

Comment:
I'd recommend changing these prop names to: `buttonText`, `description`, and `title` (since it's already the `Confirmation` component we don't need to prefix the prop names with `confirmation`)